ess/src/API/EMBC.ESS/Managers/Events/Notifications/Templates/ETransferConfirmationTemplate.hbs

```diff
@@ -0,0 +1,51 @@
+﻿<table role="presentation" style="width:94%;max-width:600px;border:none;border-spacing:0;text-align:left;font-family:Arial,sans-serif;font-size:16px;line-height:22px;color:#363636;">
+    <tbody>
+        <tr>
+            <td style="padding:30px;background-color:#ffffff;">
+                <p style='font-size: 18pt;color:#234075;font-weight: bold;'>Confirmation of e-Transfer for Emergency Support Services (ESS)</p>
+                <p><strong>Your e-Transfer request is being processed.</strong></p>
+                <p style='font-size: 13pt;color:#00a6ed;font-weight: bold;'>e-Transfer Details:</p>
+                <p>
+                    e-Transfer Recipient: <b>  {{recipientName}}</b>
+                    <br />
+                    e-Transfer Notification: <b>{{notificationEmail}}</b>
+                </p>
+                <p>
+                    {{#customComparerDecimalAmount groceryAmount 0 'Grocery'}}
```

Comment:
This part along with the helper makes no sense to me. It would be simpler, more readable and more maintainable to have the html here, the formatting as you did and check for null if to render or not